mail_template_multi_company/views/mail_template.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Acsone Sa/Nv
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="mail_template_form_view">
+        <field name="name">mail.template.form (in mail_template_multi_company)</field>
+        <field name="model">mail.template</field>
+        <field name="inherit_id" ref="mail.email_template_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='model_id']" position="after">
+                <field name="company_id" options="{'no_open': True, 'no_create': True}" groups="base.group_multi_company"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="mail_template_search_view">
+        <field name="name">mail.template.search (in mail_template_multi_company)</field>
```

Comment:
not a blocking point. ``name field`` is useless because autogenerated by Odoo by default.

Reply:
we prefer add the module name in the inherited view name, so we can easily identify it when debugging from the UI

Reply:
The module name is also easily visible in 12, because the xmlid is displayed both in the tree view and for view of ir.ui.view. So I see some value in letting Odoo computing the name itself.
Perhaps we should change the mrbob template.